Feat/landing page announcement

Avik-creator opened the pull request:
Add Landing Page

Sambit003 committed:
Removal of NX

Sambit003 committed:
init landing page

Sambit003 committed:
fix css

Sambit003 committed:
fix build dir

Sambit003 committed:
Update: Coming soon component

Sambit003 committed:
Update: Component

Sambit003 committed:
Customize Script for the removal of nx

Sambit003 committed:
Merge branch 'development' into feat/landing-page-announcement

Avik-creator committed:
Added Initial hero Section.

Avik-creator committed:
Added Full Landing Page for Now.

Avik-creator committed:
Added Changes to Fonts.

Avik-creator committed:
Added Responsiveness.

Avik-creator committed:
Added Comming Soon.

Avik-creator committed:
Added Comming Soon.

Avik-creator committed:
Removed the Rounded-SM.

Avik-creator committed:
Added Some Changes.

Avik-creator committed:
Added Changes.

Avik-creator committed:
Added Changes.

Avik-creator committed:
Merge branch 'development' into feat/landing-page-announcement

Sambit003 commented on web/src/components/ui/aurora-background.tsx:
```diff
@@ -0,0 +1,54 @@
+      >
+        <div className="absolute inset-0 overflow-hidden">
+          <div
+            //   I'm sorry but this is what peak developer performance looks like // trigger warning
```
@Avik-creator bruh 😂 fix this one

Avik-creator replied:
Bruh fixed it

Avik-creator committed:
Removed Comments.

Sambit003 commented:
One thing you can do @Avik-creator 
It's for better organization/design pattern of the react apps.

For public assets, create the types of the names in public directory for the medias, and then export-import them as type. It will have performance boost for the application. Small change, big impact. 

Inspo: https://github.com/keyshade-xyz/keyshade/tree/develop/apps/web/public

and also create a favicon of the logo from the figma file

Avik-creator committed:
Added Avatars.

Avik-creator committed:
Removed Comments.

Sambit003 commented:
@PARTHIB-DEB @Avik-creator I'm making some changes in this PR. Once done, I will merge it

Avik-creator commented:
Ok sir @Sambit003

Sambit003 committed:
Add Radix UI Avatar package and remove unused images; update AvatarRow component to prevent mutation of avatars array

Sambit003 committed:
Update layout metadata and remove unused AuroraBackgroundDemo component

- Added "Translation" to the keywords in the layout metadata for improved SEO.
- Deleted the AuroraBackgroundDemo component as it is no longer needed.

Sambit003 committed:
feat: restructure UI components and update assets

- Removed unused favicon and ComingSoon component.
- Added new hero images and shared assets for team members.
- Refactored Avatar component to improve image handling and fallback.
- Updated Hero component to use new assets and removed JoinWaitlist functionality.
- Created a new Navbar component for better header management.
- Adjusted TypeScript configuration for improved path resolution.

Sambit003 committed:
feat: add Navbar component, restructure layout, and implement global styles

Sambit003 committed:
feat: expand metadata keywords for improved SEO and discoverability

Sambit003 committed:
refactor: update global styles and configuration

- Changed global CSS file path from `src/app/globals.css` to `src/app/styles/global.css`.
- Enabled CSS variables in Tailwind configuration.
- Removed unused CSS imports and styles from `global.css`.
- Added new `next.config.ts` file to specify build directory.
- Updated import paths to use the new alias structure for better organization.
- Set background image for Hero component using a new asset.
- Adjusted Navbar and Avatar components to reflect new utility imports.

Sambit003 committed:
feat: add global styles and configuration, update layout imports

Sambit003 committed:
feat: add global CSS module declaration and update import paths for team images

Sambit003 reviewed (approved):
Made structural changes for scalability in mind. And it looks all good

Sambit003 commented on web/src/lib/constants/app.ts:
```diff
@@ -0,0 +1,58 @@
+import { type TeamMember } from '@/lib/types';
+import { SambitImg, ParthibImg, MaharshiImg, AriyanImg, AvikImg } from '@/public/team';
+
+export const TEAM_MEMBERS: readonly TeamMember[] = [
+  {
+    name: 'Sambit Chakraborty',
+    image: SambitImg,
+    alt: 'Sambit Chakraborty',
+    fallback: 'SG',
+  },
+  {
+    name: 'Parthib Kumar Deb',
+    image: ParthibImg,
+    alt: 'Parthib Kumar Deb',
+    fallback: 'PKD',
+  },
+  {
+    name: 'Maharshi Mahanti',
+    image: MaharshiImg,
+    alt: 'Maharshi Mahanti',
+    fallback: 'MM',
+  },
+  {
+    name: 'Ariyan Pandey',
+    image: AriyanImg,
+    alt: 'Ariyan Pandey',
+    fallback: 'AP',
+  },
+  {
+    name: 'Avik Mukherjee',
+    image: AvikImg,
+    alt: 'Avik Mukherjee',
+    fallback: 'AM',
+  },
+] as const;
+
```
@copilot This is not correct. Bullshit

Sambit003 commented on web/components.json:
```diff
@@ -5,16 +5,14 @@
   "tailwind": {
     "config": "tailwind.config.ts",
-    "css": "src/app/globals.css",
+    "css": "src/app/styles/global.css",
```
done

Sambit003 commented on web/public/team/index.tsx:
```diff
@@ -0,0 +1,7 @@
+import AriyanImg from './ariyan.svg'
+import AvikImg from './avik.svg'
+import MaharshiImg from './maharshi.svg'
+import ParthibImg from './parthib.svg'
+import SambitImg from './sambit.svg'
+
+export { AriyanImg, AvikImg, MaharshiImg, ParthibImg, SambitImg }
```
Not true. Ignore

Sambit003 committed:
fix: update global CSS path in Tailwind configuration
